### PR TITLE
Attempt to update Google Analytics due to deprecation of Google UA

### PR DESCRIPTION
**Pull request by jonwzheng**
See https://github.com/ReactionMechanismGenerator/RMG-website/pull/269.

This website used Google Universal Analytics to track and understand user behavior. Universal Analytics (UA) stopped working on 2023/07/01. This PR is an attempt to update to GA4 following the template provided here: https://support.google.com/analytics/answer/10271001?hl=en#zippy=%2Cin-this-article

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -133,15 +133,14 @@ <h3>
...
-<script type="text/javascript">
-    var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");
-    document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));
-</script>
-<script type="text/javascript">
-    try {
-        var pageTracker = _gat._getTracker("UA-24556433-3");
-        pageTracker._trackPageview();
-    } catch (err) {}
+<!-- Google tag (gtag.js) -->
```
Comment (mliu49) [inline index.html:136]: Seems like it's supposed to be put in the `<head>` section. I'm not sure whether it makes a difference. I think the old tag was also supposed to go in `<head>`.

**Review by mliu49 — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (jonwzheng) [reply index.html:136]: I think you're right. On the other hand it seems the tag was working previously even outside of `<head>`. Let's try it as-is, and if it doesn't work we can try moving it to `<head>`.

**Comment by jonwzheng** [comment]
Thanks!

**Comment by jonwzheng** [comment]
Can confirm it's working now.
Initial GH actions build failed because the branch settings were out of date (deploy branch was set to`master` rather than `main`).

**Pull request merged**